rhel8/profiles/stig.profile

```diff
@@ -68,167 +70,752 @@ selections:
     - enable_fips_mode
     - var_system_crypto_policy=fips
     - configure_crypto_policy
-    - configure_ssh_crypto_policy
     - configure_bind_crypto_policy
-    - configure_openssl_crypto_policy
     - configure_libreswan_crypto_policy
     - configure_kerberos_crypto_policy
     - enable_dracut_fips_module
 
-    # rules
+    ### Rules:
+    # RHEL-08-010070
     - installed_OS_is_vendor_supported
+
+    # RHEL-08-010010
     - security_patches_up_to_date
 
+    # RHEL-08-010020
     - sysctl_crypto_fips_enabled
+
+    # RHEL-08-010030
     - encrypt_partitions
+
+    # RHEL-08-010040
     - sshd_enable_warning_banner
+
+    # RHEL-08-010050
     - dconf_gnome_banner_enabled
     - dconf_gnome_login_banner_text
+
+    # RHEL-08-010060
     - banner_etc_issue
+
+    # RHEL-08-010070
+
+    # RHEL-08-010090
+
+    # RHEL-08-010100
+
+    # RHEL-08-010110
     - set_password_hashing_algorithm_logindefs
+
+    # RHEL-08-010120
+
+    # RHEL-08-010130
+    - accounts_password_pam_unix_rounds_system_auth
+    - accounts_password_pam_unix_rounds_password_auth
+
+    # RHEL-08-010140
     - grub2_uefi_password
     - grub2_uefi_admin_username
+
+    # RHEL-08-010150
     - grub2_password
     - grub2_admin_username
+
+    # RHEL-08-010151
+    - require_emergency_target_auth
+    - require_singleuser_auth
+
+    # RHEL-08-010160
+    - set_password_hashing_algorithm_systemauth
+
+    # RHEL-08-010161
     - kerberos_disable_no_keytab
+
+    # RHEL-08-010162
     - package_krb5-workstation_removed
+
+    # RHEL-08-010170
     - selinux_state
+
+    # RHEL-08-010171
     - package_policycoreutils_installed
+
+    # RHEL-08-010180
+
+    # RHEL-08-010190
+    - dir_perms_world_writable_sticky_bits
+
+    # RHEL-08-010200
     - sshd_set_idle_timeout
-    - var_sshd_set_keepalive=0
     - sshd_set_keepalive_0
+
+    # RHEL-08-010210
+
+    # RHEL-08-010220
+
+    # RHEL-08-010230
+
+    # RHEL-08-010240
+
+    # RHEL-08-010250
+
+    # RHEL-08-010260
+
+    # RHEL-08-010290 && RHEL-08-010291
+    ### NOTE: This will get split out in future STIG releases, as well as we will break
+    ### these rules up to be more flexible in meeting the requirements.
+    - harden_sshd_crypto_policy
```

Comment:
I would refrain from adding `harden_sshd_crypto_policy` at this moment since it's not really aligned with STIG.

For the Ciphers parameter it defines `aes256-ctr,aes256-cbc,aes128-ctr,aes128-cbc` whereas STIG defines `aes256-ctr,aes192-ctr,aes128-ctr` or a subset of this list.